pipeline-scripts/release.groovy

```diff
@@ -546,9 +546,9 @@ def stagePublishClient(quay_url, from_release_tag, release_name, arch, client_ty
                 source_name="openshift-client"
                 ;;
               installer)
-                source_name="openshift-installer"
+                source_name="openshift-install"
                 ;;
-              operator-registry)
+              operator-framework-olm)
```

Comment:
We don't have `operator-framework-olm` in the release, did you mean to set the `source_name` ?

Reply:
```
oc adm release info 4.12.13 --output=json | jq -r '.references.spec.tags[] | select (.name=="operator-registry") | .annotations'
{
  "io.openshift.build.commit.id": "dd3cc9cbf5ccc16441f427b2d1386d6ca430f82f",
  "io.openshift.build.commit.ref": "",
  "io.openshift.build.source-location": "https://github.com/openshift/operator-framework-olm"
}
```

It's the "operator-registry" image that is used to determine the SHA of the opm source code in the "operator-framework-olm" repo: https://github.com/openshift/operator-framework-olm/tree/master/staging/operator-registry/cmd/opm

So in this case, the "operator-registry" image will say it got the source from the "operator-framework-olm" repo, and we are swapping that our here with the "opm" source name so we are writing the file correctly

Reply:
Ahh my bad, just had a look at your earlier PR. Looks good.